source/SpriteParameters.h

```diff
@@ -0,0 +1,126 @@
+/* SpriteParameters.h
+Copyright (c) 2024 by XessWaffle
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef SPRITE_PARAMETERS_H_
+#define SPRITE_PARAMETERS_H_
```

Comment:
we switched to `#pragma once` a while ago
```suggestion
#pragma once
```